apps/kyc-server/src/components/dashboard/table/user-table-columns.tsx

```
@@ -0,0 +1,126 @@
+import type { ColumnDef } from '@tanstack/react-table'
+import { ShieldCheckIcon, UserCheckIcon, UserIcon } from 'lucide-react'
+
+import { Badge } from '~/components/base/badge'
+import { Checkbox } from '~/components/base/checkbox'
+import { DragHandle } from '~/components/dashboard/table/drag-handle'
+import { UserActionsMenu } from '~/components/dashboard/table/user-actions-menu'
+import { UserDetailsSheet } from '~/components/dashboard/table/user-details-sheet'
+import { getStatusColor, getStatusIcon } from '~/utils/table'
+
+type UserData = any
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)